canonical-websites/www.ubuntu.com
# 2878 robotics updates

## Done

Update copy on internet of things pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/internet-of-things/robotics>
- Make sure page matches [copy doc](https://docs.google.com/document/d/1g8XpVzDkAdFETrfZvOf5Q6WlYwHCBYQ8Y6TgTlyTEYc/edit#)
- View the site locally in your web browser at: <http://0.0.0.0:8001/internet-of-things/digital-signage>
- Make sure page matches [copy doc]()
- `/internet-of-things/gateways` was done in a previous PR

@anasereijo Frankie asked if you could review the logo grid under the 'Ubuntu runs on the robotic platforms that matter' heading on /internet-of-things/robotics as we removed a logo

## Issue / Card

Fixes #2878
